hermes-console/static/js/console/topic/TopicController.js

```diff
@@ -224,6 +224,17 @@ topics.controller('TopicController', ['TOPIC_CONFIG', 'TopicRepository', 'TopicM
                 loadSubscriptions();
             });
         };
+
+        $scope.copyToClipboard = function () {
+            var tempElement = document.createElement('textarea');
+            tempElement.value = $scope.messageSchema;
+            document.body.appendChild(tempElement);
+            tempElement.select();
+            document.execCommand('copy');
+            document.body.removeChild(tempElement);
+
+            toaster.pop('info', 'Info', 'Message scheme has been copied to clipboard');
```

Comment:
Message `schema` has been copied to clipboard

Reply:
true, fixed